tests/testBurrow.ml

```diff
@@ -644,6 +752,46 @@ let suite =
             (Ligo.mul_int_int lp_den f_den))
     );
 
+    (* TODO: Property "compute_tez_to_auction - <= 0 for underburrowed burrows" *)
+    (* TODO: Property "compute_tez_to_auction - > 0 for overburrowed burrows" *)
```

Comment:
I'd hedge or even remove these two comments until we are sure that these properties hold (or not).

Reply:
Good eye, I had actually thought I had already removed these.